tools/StaticAnalysis/packages.config

```diff
@@ -1,7 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="CommandLineParser" version="1.9.71" targetFramework="net45" />
+  <package id="Hyak.Common" version="1.0.3" targetFramework="net452" />
   <package id="log4net" version="2.0.5" targetFramework="net45" />
+  <package id="Microsoft.ApplicationInsights" version="1.2.0" targetFramework="net452" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.2" targetFramework="net452" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.PowerShell.Authentication.Abstractions" version="1.0.0" targetFramework="net452" />
```

Comment:
??

Reply:
Added since we no longer depend on the `Commands.Common` project.